staging/src/k8s.io/kube-scheduler/config/v1beta1/types.go

```diff
@@ -44,56 +44,58 @@ const (
 type KubeSchedulerConfiguration struct {
 	metav1.TypeMeta `json:",inline"`
 
-	// Parallelism defines the amount of parallelism in algorithms for scheduling a Pods. Must be greater than 0. Defaults to 16
+	// The amount of parallelism in algorithms for scheduling Pods.
+	// Must be greater than 0. Defaults to 16.
 	Parallelism *int32 `json:"parallelism,omitempty"`
 
-	// LeaderElection defines the configuration of leader election client.
+	// The configuration of leader election client.
 	LeaderElection componentbaseconfigv1alpha1.LeaderElectionConfiguration `json:"leaderElection"`
 
-	// ClientConnection specifies the kubeconfig file and client connection
-	// settings for the proxy server to use when communicating with the apiserver.
+	// This specifies the kubeconfig file and client connection settings for the scheduler
+	// when communicating with the API server.
 	ClientConnection componentbaseconfigv1alpha1.ClientConnectionConfiguration `json:"clientConnection"`
-	// HealthzBindAddress is the IP address and port for the health check server to serve on,
-	// defaulting to 0.0.0.0:10251
+	// The IP address and port for the health check server to serve on,
+	// defaulting to "0.0.0.0:10251".
 	HealthzBindAddress *string `json:"healthzBindAddress,omitempty"`
-	// MetricsBindAddress is the IP address and port for the metrics server to
-	// serve on, defaulting to 0.0.0.0:10251.
+	// The IP address and port for the metrics server to serve on,
+	// defaulting to "0.0.0.0:10251".
 	MetricsBindAddress *string `json:"metricsBindAddress,omitempty"`
 
-	// DebuggingConfiguration holds configuration for Debugging related features
+	// The configuration for debugging related features.
 	// TODO: We might wanna make this a substruct like Debugging componentbaseconfigv1alpha1.DebuggingConfiguration
```

Comment:
do we want todo's to be part of the public docs? I think we can remove this one and just keep the todo in the internal type.

Reply:
Right. Will do.